components/switch/switch.js

```diff
@@ -6,7 +6,7 @@ import { SwitchMixin } from './switch-mixin.js';
 
 /**
  * A generic switch with on/off semantics.
- * @attr {string} text - REQUIRED: The text that is displayed for the switch label.
+ * @attr {string} text - ACCESSIBILITY: REQUIRED: Acts as the [primary label](https://www.w3.org/WAI/tutorials/forms/labels/) for the switch. Visible unless text-position is `hidden`.
```

Comment:
Hopefully it's obvious that by providing `text`, we handle the label for them, and that they don't need to wrap their switch inside of a `<label>`, add `aria-label` or one of the techniques listed in the W3C forms doc. If we really want to link to something here, should we be limiting to the WCAG criteria instead?

Reply:
I feel like it's straightforward since it's saying the text provided will act as the label. We could specify that they shouldn't need to apply `aria-label` to the switch, but I'm not entirely sure how beneficial that information would be. In terms of changing the link, would this be the preferred one? https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html Or did you have a particular one in mind?